apps/api/src/app/partner-integrations/usecases/set-vercel-configuration/set-vercel-configuration.usecase.ts

```diff
@@ -10,24 +11,24 @@ import { SetVercelConfigurationCommand } from './set-vercel-configuration.comman
 export class SetVercelConfiguration {
   constructor(
     private httpService: HttpService,
-
     private organizationRepository: OrganizationRepository
   ) {}
 
   async execute(command: SetVercelConfigurationCommand): Promise<SetupVercelConfigurationResponseDto> {
     try {
       const tokenData = await this.getVercelToken(command.vercelIntegrationCode);
 
-      if (!tokenData) throw new ApiException('No token data found');
-
-      const saveConfigurationData = {
+      const configuration = {
         accessToken: tokenData.accessToken,
         configurationId: command.configurationId,
-        teamId: tokenData.teamId as string,
+        teamId: tokenData.teamId,
         partnerType: PartnerTypeEnum.VERCEL,
       };
 
-      await this.saveConfiguration(command.organizationId, command.userId, saveConfigurationData);
+      await this.organizationRepository.upsertPartnerConfiguration({
+        organizationId: command.organizationId,
+        configuration,
+      });
```

Comment:
Create a single partner configuration for the current org. Before we were creating for all user orgs.